tests/installation/installation_overview.pm

```diff
@@ -48,10 +48,16 @@ sub run {
     my ($self) = shift;
     # overview-generation
     # this is almost impossible to check for real
-    assert_screen "installation-settings-overview-loaded", 150;
-    $self->deal_with_dependency_issues;
-    assert_screen "inst-xen-pattern" if get_var('XEN');
-    ensure_ssh_unblocked;
+    if (is_caasp && check_var('HDDSIZEGB', '10')) {
+        # boo#1099762
+        assert_screen('installation-settings-overview-loaded-impossible-proposal');
```

Comment:
I suggest to just make one common call to assert_screen but add tags selectively